pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -172,6 +173,18 @@ type CSINodeInfo interface {
 	GetCSINodeInfo(nodeName string) (*storagev1beta1.CSINode, error)
 }
 
+var _ CSINodeInfo = &CachedCSINodeInfo{}
+
+// CachedCSINodeInfo implements CSINodeInfoInfo
+type CachedCSINodeInfo struct {
+	storagev1beta1listers.CSINodeLister
```

Comment:
why even wrap this one interface inside this struct, why not just use the `CSINodeLister` interface directly

*this comment superscedes others*

Reply:
I am planning a cleanup PR for all "listers/getters" the scheduler defines, so lets do that there. What I wanted to do in this PR is just cleanup the cache and maintain the existing interface that other parts of the code depend on (and hence I had to define this interface temporarily).